src/components/Popup/MainHeader.vue

```diff
@@ -11,6 +11,15 @@
       >
         <IconCog />
       </div>
+      <div
+        class="icon"
+        id="i-plus"
+        v-bind:title="i18n.settings"
```

Comment:
Alt text has the wrong string

Reply:
I've updated the string, I've added `Add code`.